### ISSUE TITLE
Testing Actions in Vuex Modules

### ISSUE THREAD

**Issue by mabumusa1** [vuejs/vuex#775]
### Version
2.3.0

### Reproduction link
[https://jsfiddle.net/chrisvfritz/50wL7mdz/](https://jsfiddle.net/chrisvfritz/50wL7mdz/)

### Steps to reproduce
1. Make any module for a store
2. Add Any Action that uses getter
3. Try to access context.getters it will be undefined

### What is expected?
To find the getters to be defined with the context of the test

### What is actually happening?
the getters is undefined because the helper function provided in the documentation does not pass the getters

---
I tried to pass getters and use the same method used in the documentation but that did not work for me

**Comment by posva** [comment]
Provide a real repro, please

**Comment by mabumusa1** [comment]
I spent sometime analyzing this issue, here is a better summary and repo attached at the bottom. 

According to the [testing guides](https://vuex.vuejs.org/en/testing.html), section of **Testing Actions** I can use the helper function to mock actions, but this mock does not pass the getters therefore I can not use the statement `context.getters` which is mentioned [here](https://vuex.vuejs.org/en/actions.html).

I tried to patch the code to pass getters to the mock function, then I realized that I have to pass so `payload` for `getters` in order to make them work, something similar to testing getters mentioned in the same testing guidelines. That approach did not work for me.

I decided not use the getters inside the `action` now and pass it to the `action` instead, I think it is not a priority at the meantime to mock getters since they can be passed to actions, but it would be nice to have in case it was used inside an action as in my case.

[Repo of the code](https://github.com/mabumusa1/studious-meme/tree/master)

I will close this issue at this time, thanks lots of your help and building this awesome technology
